applications/FemToDemApplication/custom_elements/base_solid_element.h

```diff
@@ -985,6 +985,11 @@ class BaseSolidElement
             // Compute material reponse
             this->SetConstitutiveVariables(this_kinematic_variables, this_constitutive_variables, Values, point_number, integration_points);
 
+            // rotate to local axes strain/F
```

Comment:
Question, any plan to unify the elements with the structural in your app?

Reply:
Im gonna revert this, was a mistake!